docs/intro/from-ground-up.rst

```diff
@@ -160,12 +160,12 @@ The same, but using web-poet
     # === Framework-specific I/O code
     def download_sync(url):
         resp = requests.get(url)
-        return ResponseData(url=resp.url, html=resp.text)
+        return HttpResponse(url=resp.url, body=resp.content)
```

Comment:
this needs to be a bit more elaborate, we need to pass headers as well, in order for the response to be decoded properly

Reply:
See also: https://github.com/scrapinghub/web-poet/commit/92f114cd1c7294e18f70ef76f2b670a74bdb3982 (the commit is in a branch, it doesn't belong to any PR)

Reply:
Done! https://github.com/scrapinghub/web-poet/pull/31/commits/7125ec7b8a027a37d58242847de975b14ff0839f